README.md

```diff
@@ -8,10 +8,18 @@ Tekton Dashboard is a general purpose, web-based UI for Tekton Pipelines. It all
 
 ## Getting Started
 
-[coming soon] To deploy the dashboard, execute the following command:
-
+**Currently** to view the backend APIs, at `localhost:9087/v1/namespaces/<namespace>/<resource name>` for example:
+  
```

Comment:
Lets not forget you currently still need to build/push and alter the yaml before the apply

Reply:
D'oh, will fix 😄